andela/anbu-rc
# #137005169 Embed social content

#### What does this PR do?
This PR embeds disqus for products review.
#### Description of Task to be completed?
Embed reviews, tweets, and other social content.
#### How should this be manually tested?
On the website, click on a product to see its details and you will be able to comment on the product via disqus.
#### Any background context you want to provide?
None.
#### What are the relevant pivotal tracker stories?
#137005169.
#### Screenshots (if appropriate)
![embed disqus](https://cloud.githubusercontent.com/assets/23452132/22010850/ffbe9164-dc8b-11e6-86dc-0828d1bf1801.png)
#### Questions:
None.
